--- a/sw-update.sh	2017-09-21 09:51:48.906103000 +0200
+++ b/sw-update.sh	2017-09-01 11:10:19.346543000 +0200
@@ -136,7 +136,7 @@ flash_firmware()
   fi
 
   log INFO "Formatting partition ${DEST_PARTITION} ..."
-  mkfs.ext4 ${DEST_PARTITION}
+  mkfs.ext4 -F ${DEST_PARTITION}
   if [ "$?" != "0" ]
   then
     log ERROR "Formating parition ${DEST_PARTITION} failed"
@@ -230,7 +230,7 @@ update_linux()
   log INFO "Installing linux update ..."
 
   flash_firmware
-  cleanup
+  # cleanup
 
   log INFO "Linux update succeed!"
 }
